masterlist.yaml

```
@@ -12483,6 +12511,17 @@ plugins:
         crc: 0x00AFAC80
         util: '[SSEEdit v4.0.3](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
         itm: 3
+    
+  - name: 'Falskaar - Bruma Patch.esp'
+    msg:
+      - <<: *useInstead
+        subs: [ '[Legacy of the Dragonborn Patches (Official)](https://www.nexusmods.com/skyrimspecialedition/mods/30980/)' ]
+    
+  - name: 'Falskaar - Unique Region Names.esp'
+    msg:
+      - <<: *useInstead
+        subs: [ '[Unique Region Names](www.nexusmods.com/skyrimspecialedition/mods/11750/)' ]
```

```suggestion
        subs: [ '[Unique Region Names - Falskaar addon](www.nexusmods.com/skyrimspecialedition/mods/11750/)' ]
```
Again, just to make it clear what's being recommended.